packages/styles/code.css

```diff
@@ -88,11 +88,11 @@
 .Code .hljs-subst,
 .Code .hljs-meta,
 .Code .hljs-link {
-  color: #f99157;
+  color: #8a3500;
```

Comment:
🥴 the difference is jarring
<img width="418" alt="Screenshot 2023-05-09 at 1 33 32 PM" src="https://github.com/dequelabs/cauldron/assets/3180185/57515190-1ca7-49fa-89f2-e1b82ec89758">

Reply:
I know, but that's that @dequelabs/cauldron-design wanted 🤷